src/unitxt/deprecation_utils.py

```diff
@@ -98,3 +98,14 @@ def decorator(obj):
         return depraction_wrapper(func, version, alt_text)
 
     return decorator
+
+
+def init_warning(initiated_class):
```

Comment:
```suggestion
def init_warning(initiated_class, msg=""): 
```
Can you add an option to add a custom message?